app_book/forms.py

```diff
@@ -16,12 +16,23 @@ class BookForm(forms.ModelForm):
     widgets: Defintion wie die Eingabefelder auszusehen haben. Hier am Beispiel css Klasse von Bootstrap genutzt und Vorschau eingebaut
     labels: Definition was bei den Label Tags auf der Oberflaeche erscheinen soll. Wenn dies nicht definiert worde ist wird der Attributenname der Modellklasse genommen
     '''
+    delete_saved_image = forms.BooleanField(required=False, label='Gespeichertes Bild löschen')
```

Comment:
Ich wollte den Haken erst zum Bild-Widget hinzufügen, habe diesen dann als eigenständiges Formularfeld eingefügt - mein erster Ansatz wirkte etwas ghettohaft in der Umsetzung. Mit dem Formularfeld fand ich es einfacher, es bedingt darzustellen und zu nutzen. Weiß allerdings nicht, wie ich Formularfelder umpositionieren kann.